src/System.Private.CoreLib/shared/System/Text/UTF32Encoding.cs

```diff
@@ -42,27 +42,27 @@ public sealed class UTF32Encoding : Encoding
         private static readonly byte[] s_bigEndianPreamble = new byte[4] { 0x00, 0x00, 0xFE, 0xFF };
         private static readonly byte[] s_littleEndianPreamble = new byte[4] { 0xFF, 0xFE, 0x00, 0x00 };
 
-        private bool _emitUTF32ByteOrderMark = false;
-        private bool _isThrowException = false;
-        private bool _bigEndian = false;
+        private readonly bool _emitUTF32ByteOrderMark;
```

Comment:
It does not help to delete `= false` initializations of instance fields since Roslyn will optimize these out anyway. Folks tend to write the initializations for clarity.

Reply:
I don't know what is coding style on this in CoreCLR as each file does it differently but I'd argue that this is actually less clear. Compare

```c#
class X
{
        private readonly bool _valueA = false;
        private readonly bool _valueB = true;
        private readonly bool _valueC = false;

		public X (bool value)
		{
			this._valueC = value;
		}
}
```
with

```c#
class X
{
        private readonly bool _valueA;
        private readonly bool _valueB = true;
        private readonly bool _valueC;

		public X (bool value)
		{
			this._valueC = value;
		}
}
```

but could add the redundant initialization

Reply:
There is no written down convention for this style detail (https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md).

We typically prefer to keep the subjective style details like this alone since the next person has a different opinion and changes it back.